zeroize/src/lib.rs

```diff
@@ -580,6 +580,26 @@ where
     }
 }
 
+impl<T: ?Sized, Z> AsRef<T> for Zeroizing<Z>
+where
+    Z: Zeroize,
+    Z: AsRef<T>,
```

Comment:
I think it's a bit weird to have some bounds inline and some in the where clause.

```suggestion
impl<T, Z> AsRef<T> for Zeroizing<Z>
where
    T: ?Sized,
    Z: Zeroize,
    Z: AsRef<T>,
```

Reply:
Fixed.